crates/pep508-rs/src/marker/tree.rs

```diff
@@ -2160,14 +2160,14 @@ mod test {
             simplified("python_version == '3.8'")
                 .try_to_string()
                 .unwrap(),
-            "python_version == '3.8'"
+            "python_full_version < '3.9'"
```

Comment:
This one was surprising out of context (and a similar change in the transformers test). I understand this is due to the distributions Python requirement providing a lower bound?

Reply:
Yes, it would have been `python_full_version >= 3.8 and python_full_version < '3.9'`. We could simplify this to `python_version == 3.8.*`, but this just ends up happening because of the order in which we try transformations (removing lower bounds before attempting to convert to star equality).